secrets/awskms/testdata/TestConformance/TestMultipleEncryptionsNotEqual.yaml

```diff
@@ -10,22 +10,22 @@ interactions:
       Content-Type:
       - application/x-amz-json-1.1
       User-Agent:
-      - aws-sdk-go/1.15.57 (go1.11.2; linux; amd64)
+      - aws-sdk-go/1.16.23 (go1.11.2; linux; amd64)
```

Comment:
Nit: I don't think you needed to modify these replay files in this PR, just add your new one.